MLTAscend/MLTAscend.Data/Helpers/PredictionHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using dom = MLTAscend.Domain.Models;
+using System.Linq;
+
+
+namespace MLTAscend.Data.Helpers
+{
+    public class PredictionHelper
+    {
+        private static MLTAscendDbContext _db = new MLTAscendDbContext();
+
+        public dom.Prediction GetPredictionByTicker(string ticker)
+        {
+            return _db.Predictions.FirstOrDefault(m => m.Ticker == ticker);
+        }
+
+        public bool SetPrediction(dom.Prediction prediction)
+        {
+            _db.Predictions.Add(prediction);
```

Comment:
I'm not sure whether the discrepancy is with this class or the UserHelper, but in the UserHelper you set the creation date in your "SetUser" method

Reply:
If it's intended in some way or solved through the database, let me know

Reply:
oh yeah i forgot to add it to that one

i can handle that now, doing the set datetimenow alleviates the issue of timezones too